packages/devtools_app/lib/src/shared/landing_screen.dart

```diff
@@ -259,15 +259,14 @@ class ImportFileInstructions extends StatelessWidget {
     final importedFile = await importFileFromPicker(
       acceptedTypes: ['json'],
     );
-    if (importedFile != null) {
-      Provider.of<ImportController>(context, listen: false)
-          .importData(importedFile);
-    }
+
+    Provider.of<ImportController>(context, listen: false)
```

Comment:
importFileFromPicker can return null, so we'll want to keep this null check here.

Reply:
done